engagement_db_to_analysis.py

```diff
@@ -21,18 +21,21 @@
     parser.add_argument("configuration_module",
                         help="Configuration module to import e.g. 'configurations.test_config'. "
                              "This module must contain a PIPELINE_CONFIGURATION property")
+    parser.add_argument("production_csv_output_path", metavar="production-csv-output-path",
```

Comment:
can this be an analysis_dir instead? I think allowing this to be specified separately is unnecessary and makes setting up pipelines slower.